dashboard/app/models/user_permission.rb

```diff
@@ -54,7 +54,7 @@ class UserPermission < ApplicationRecord
     AUTHORIZED_TEACHER,
   ].freeze
 
-  ASSIGNABLE_PERMISSIONS = (VALID_PERMISSIONS - [WORKSHOP_ORGANIZER]).freeze
+  ASSIGNABLE_PERMISSIONS = (VALID_PERMISSIONS).freeze
```

Comment:
do we want to remove this and use `VALID_PERMISSIONS`? It looks like only https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/views/admin_users/permissions_form.html.haml would need to be updated (in two places)